Portfolio/index.html

```diff
@@ -4,6 +4,7 @@
     <title>Portfolio</title>

```

Comment:
Consider moving the link to the CSS file to the head section of the HTML document for better performance.